### PR TITLE
Add metrics anomaly detection content

### PR DESCRIPTION
**Pull request by EamonnTP**
This PR adds the metrics anomaly detection content to the Observability Guide.

### HTML preview
https://observability-docs_167.docs-preview.app.elstc.co/guide/en/observability/master/inspect-metric-anomalies.html

### Related issue
Closes https://github.com/elastic/observability-docs/issues/137

### CODE REVIEW

File: docs/en/observability/inspect-metric-anomalies.asciidoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Create a {ml} job to detect anomalous memory usage and network traffic automatically.
+
+Once the {ml} jobs are created, these settings can not be changed. You can recreate
+these jobs later. However, any previously detected anomalies are removed.
+
+1. In the side navigation, expand *Observability*, click *Metrics*, and then click *Anomaly detection*.
+2. You’ll be prompted to create a machine learning job for *Hosts* or *Kubernetes Pods*.
+3. Choose a start date for the machine learning analysis.
++
+Machine learning jobs analyze the past four weeks of data and continue to run indefinitely.
```
Comment (szabosteve) [inline docs/en/observability/inspect-metric-anomalies.asciidoc:25]:
```suggestion
Machine learning jobs analyze the last four weeks of data and continue to run indefinitely.
```

Comment (szabosteve) [reply docs/en/observability/inspect-metric-anomalies.asciidoc:25]: Small change just to be in-line with the UI text that has been changed via https://github.com/elastic/kibana/pull/79425

File: docs/en/observability/inspect-metric-anomalies.asciidoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+You can specify a different start date, end date, or both, but we recommend that you run jobs
```
Comment (szabosteve) [inline docs/en/observability/inspect-metric-anomalies.asciidoc:26]: I cannot specify the end date when creating the job. Is it really possible?

Comment (EamonnTP) [reply docs/en/observability/inspect-metric-anomalies.asciidoc:26]: Just checked the latest build and you're correct. You can only specify the start date.

File: docs/en/observability/inspect-metric-anomalies.asciidoc (hunk 1 above)
Comment (EamonnTP) [reply docs/en/observability/inspect-metric-anomalies.asciidoc:25]: Thanks.

File: docs/en/observability/inspect-metric-anomalies.asciidoc [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+When the {anomaly-detect} features of {ml} are enabled,
+you can create {ml} jobs to detect and inspect memory usage
+and network traffic anomalies for hosts and Kubernetes pods.
+
+You can model system memory usage, along with inbound and outbound
+network traffic across hosts or pods. You can detect unusual
+increases in memory usage, and unusually high inbound or outbound traffic
+across hosts or pods.
+
+[ml-jobs-hosts]]
```
Comment (szabosteve) [inline docs/en/observability/inspect-metric-anomalies.asciidoc:13]:
```suggestion
[[ml-jobs-hosts]]
```

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**